recipes/Cfg/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cfg" %}
+{% set version = "9.0.0" %}
+{% set tag = "v9r0" %}
+
+package:
+   name: "{{ name|lower }}"
+   version: "{{ version }}"
+
+source:
+   url: https://git.ligo.org/virgo/virgoapp/Cfg/-/archive/{{ tag }}/Cgf-{{ tag }}.tar.gz
```

Comment:
@rhyspoulton, the following might be a good change to make to not duplicate the version number/tag for each release:

```suggestion
{% set version = "v9r0" %}

# handle Virgo version number format
{% set semver_version = version|
    replace('v', '', 1)|
    replace('r', '.', 1)|
    replace('p', '.', 1)
 %}

package:
  name: {{ name|lower }}
  version: {{ semver_version }}

source:
  url: https://git.ligo.org/virgo/virgoapp/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
```

(not that I fixed a typo in the tarball URL: `Cgf`)

Reply:
Yes, this update makes sense, I take it that this will need to be an update since it is already merged?

Reply:
```suggestion
{% set name = "cfg" %}
{% set version = "v9r0" %}

# handle Virgo version number format
{% if 'p' in version %}
  {% set semver_version = version|
      replace('v', '', 1)|
      replace('r', '.', 1)|
      replace('p', '.', 1)
  %}
{% else %}
  {% set semver_version = version|
      replace('v', '', 1)|
      replace('r', '.', 1)
      + '.0'
  %}
{% endif %}

package:
  name: "{{ name|lower }}"
  version: "{{ semver_version }}"
source:
  url: https://git.ligo.org/virgo/virgoapp/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
```

I needed to add an if-else since when there is `p0` in the Virgo version this is often just dropped. So in the case when `p` does not exist in the Virgo version then just add a `.0` to the version number.

Reply:
I would personally not include the `if-else` statement, since some packages do ship versions without a trailing patch version `.0`. If they want to include a patch version, they should include it in the upstream tag.

Reply:
When I use the original (no `if-else`) version I get the following error when the version check is done (this is the main reason why I added the `if-else`):
`pkg-config --print-errors --exact-version 9.0 cfg`
`Requested 'cfg = 9.0' but version of cfg is 9.0.0`
It seems to be dependent on how the version defined in the `.pc` file (which in the case of Virgo software gets it from the `meson.build` or `CMakeLists.txt` file).
I am not sure of the best way to deal with this, any ideas?

Reply:
Ok, so the authors themselves are inconsistent between the tag and the semantic version number, great. What you have is the best solution then.